assistant/db/prompt_query.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+
+from sqlite3 import Connection
+
+
+def get_stable_prompt(conn: Connection, language: str):
+    query = """
+        SELECT *
+        FROM prompt_detail pd
+        LEFT JOIN prompt p
+        ON pd.prompt_id == p.id
+        WHERE p.stable == 1
```

Comment:
Using a stable flag column means you can have duplicates. Either make this its own mini-table, or sort the result by creation date and use the newest-stable.

Reply:
This is solved in the seeder, I misunderstood.